content/blog/2021-01-20-january-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,168 @@
+description: |
+  Monthly updates are here- read all about 
+  our new R language tutorial, putting DVC 
+  to work on an image segmentation pipeline,
+  and a new fast way to setup your DVC remote.
+descriptionLong: |
+  Monthly updates are here- read all about 
+  our new R language tutorial, putting DVC 
+  to work on an image segmentation pipeline,
+  and a new fast way to setup your DVC remote.
+picture: 2020-12-18/cover.png
+pictureComment: |
+  This holiday season, show your loved ones
+  you care with our new shirt.
+
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/dvc-heartbeat-jan-21/632
+tags:
+  - Heartbeat
+  - CML
+  - DVC
+  - DAGsHub
+  - R
+  - MLOps
+---
+
+## News
+
+Welcome to the first Heartbeat of 2021! Here's some new year news.
+
+### We're still hiring
+
+Our search continues for a
+[**Developer Advocate**](https://weworkremotely.com/remote-jobs/iterative-developer-advocate)
+to support and inspire developers by creating new content like blogs, tutorials,
+and videos- plus lead outreach through meetups and conferences.
+
+Does this sound like you or someone you know? Be in touch!
+
+### 7000 stars on GitHub
+
+We recently passed 7000 stars on the
+[DVC GitHub repository](https://github.com/iterative/dvc)! We crossed the 7k
+mark extremely close to midnight on New Year's Eve, so we probably hit it in
+time for the new year in at least one time zone. Anyway, it made for a very
+suspenseful countdown to midnight. Woot woot!
+
+https://media.giphy.com/media/QAPFLCrpfalPi/giphy.gif
+
+The repo is HQ for DVC development, meaning- if you have an issue to report, a
+feature to request, or a pull request to offer, this is where you should start!
+
+### New video for R users
+
+A lot of our videos about GitHub Actions have used Python scripts, but there's
+no reason to restrict [Continuous Machine Learning](https://cml.dev) to one
+language. We've just released our first-ever R language video, which covers
+
+- How to install R on a GitHub Actions runner
+- How to manage R package dependencies for continuous integration (teaser: CRAN
+  binaries are amazing)
+- Putting a `ggplot` or a `kable` table in your pull request
+
+Watch and follow along! If you make something based on this approach, or if you
+think there's a better way, please tell us- we're eager to see what the R
+community thinks.
+
+https://youtu.be/NwUijrm2U2w
+
+### Workshops and talks
+
+Coming up Friday, Jan 21, I (Elle) will be speaking with
+[Alexey Grigorev](https://twitter.com/Al_Grigor), author of a
+[Data Science Bookcamp](https://mlbookcamp.com/), on his podcast about being a
+developer advocate in the machine learning space! If you're curious about what
+the role entails, or what to look for when hiring a developer advocate for your
+machine learning project, please come by. The event will be recorded live and
+then shared for your listening pleasure 🎧
+
+https://twitter.com/Al_Grigor/status/1351955523890012167
+
+## From the community
```

Comment:
Oh, I see the second H2 here, sorry about the previous comment - you can disregard.